src/xcode/ENA/ENA/Source/Extensions/__tests__/PDFDocument+EmbedImageTests.swift

```diff
@@ -0,0 +1,27 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+import PDFKit
+@testable import ENA
+
+class PDFDocument_EmbedImageTests: XCTestCase {
+	
+	// swiftlint:disable force_unwrapping
+	/// Not proud of this test, if you have a nicer idea, please go ahead.
+    func testEmbedingImage() throws {
```

Comment:
Indentation looks weird? Is this Github, again?

Reply:
Yep Github at it again